joblib/func_inspect.py

```diff
@@ -11,11 +11,15 @@
 import warnings
 import re
 import os
-
+import sys
 from ._compat import _basestring
 from .logger import pformat
 from ._memory_helpers import open_py_source
 
+
+PY3 = sys.version_info[0] >= 3
```

Comment:
The same global variable is used in a lot of other places. I was thinking of moving them to compat.py, in order to have defined in a single place.